### PR TITLE
[cQubeChat] Week 1 and Week 2 updates

### PR DESCRIPTION
**Pull request by suyashgautam**
## Project

cQubeChat

### Changes

Week 1 and Week 2 updates added

### Notes

Currently, I am working on presenting a demo on the 17th. Once that is done I will create PRs and will update the same.

### CODE REVIEW

File: docs/2023/cQube/cQubeChat/updates/2023-07-07.md [hunk 1]
```diff
@@ -4,13 +4,22 @@ author: Suyash Gautam
...
+
 ## Contributions
 
+- https://github.com/Sunbird-cQube/cQubeChat/issues/10
+- https://github.com/Sunbird-cQube/cQubeChat/issues/7
+
 ## Learnings
+
+SQLDatabaseChain is good for cases where we have we have small database and interactive table names.
+Streamlit provides ways to create chart which will become useful later on to show charts for the data.
```
Comment (claddyk) [inline docs/2023/cQube/cQubeChat/updates/2023-07-07.md:25]: nit:
```bash
- SQLDatabaseChain is good for cases where we have we have small database and interactive table names.
- Streamlit provides ways to create chart which will become useful later on to show charts for the data.
```

Comment (claddyk) [inline docs/2023/cQube/cQubeChat/updates/2023-07-07.md:25]: nit:
```bash

- SQLDatabaseChain is good for cases where we have we have small database and interactive table names.

- Streamlit provides ways to create chart which will become useful later on to show charts for the data.
```

**Review by claddyk — CHANGES_REQUESTED** [review]

File: docs/2023/cQube/cQubeChat/updates/2023-07-07.md (hunk 1 above)
Comment (suyashgautam) [reply docs/2023/cQube/cQubeChat/updates/2023-07-07.md]: I have fixed the issue in the latest commit

Comment (claddyk) [reply docs/2023/cQube/cQubeChat/updates/2023-07-07.md]: Looks good, but generally you should add the reference to your commit in your comment.
It's just a good practice.

**Pull request merged**